cmake/Platform/Linux/LYWrappers_linux.cmake

```diff
@@ -6,6 +6,15 @@
 #
 #
 
+# Check if llvm-strip is available before we can set the option to strip debug symbols from output binaries
+find_program(LLVM_STRIP_TOOL llvm-strip)
+if (LLVM_STRIP_TOOL)
+    set(LY_STRIP_DEBUG_SYMBOLS FALSE CACHE BOOL "Flag to strip debug symbols from the (non-debug) output binaries")
+else()
+    message(WARNING "Unable to locate 'llvm-strip' tool needed to strip debug symbols from the output target(s). Debug symbol "
+                    "stripping (LY_STRIP_DEBUG_SYMBOLS) will not be supported. Make sure the llvm tools are installed.")
+endif()
```

Comment:
The logic sends a warning before checking if a user wants to strip debug symbols at all.

It should only warn if `LY_STRIP_DEBUG_SYMBOLS` is true AND either llvm-strip or GNU strip is missing.